src/desktop/apps/notifications/client/index.coffee

```diff
@@ -1,7 +1,6 @@
 _ = require 'underscore'
 Backbone = require 'backbone'
 { FOLLOWING, API_URL } = sd = require('sharify').data
-scrollFrame = require 'scroll-frame'
```

Comment:
Things removed in this file were made obsolete when we created a React version of the desktop view.